lib/modules/modules.go

```diff
@@ -162,11 +168,14 @@ func (p *defaultModules) PrintVersion() {
 
 // Features returns supported features
 func (p *defaultModules) Features() Features {
+	automaticUpgrades, _ := utils.ParseBool(os.Getenv(defaults.AutomaticUpgradesEnvar))
```

Comment:
Tbh, instead of silently ignoring the error, I would handle the case with the unset env. var explicitly and log the error from ParseBool. This will help in troubleshooting if we accidentally set incorrect value for example.